applications/benchmark/BuildGradient.cpp

```diff
@@ -373,17 +182,22 @@ BENCHMARK(BuildGradient, NuToPDE, runner)
     NuTo::ElementFem displElement(displNodes, displInterpolation);
 
     NuTo::DofType displDof("Displacements", 2);
-    NuTo::ElementCollection elements(coordElement);
-    elements.AddDofElement(displDof, displElement);
+    NuTo::ElementCollectionFem element(coordElement);
+    element.AddDofElement(displDof, displElement);
 
     NuTo::Laws::LinearElastic<2> law(20000, 0.3, NuTo::ePlaneState::PLANE_STRAIN);
-    NuTo::Integrands::TimeDependent::MomentumBalance<2> integrand(displDof, law);
+    NuTo::Integrands::MomentumBalance<2> integrand(displDof, law);
     const NuTo::IntegrationTypeTensorProduct<2> it(2, NuTo::eIntegrationMethod::GAUSS);
 
-    NuTo::Cell cell(elements, it, integrand);
+    NuTo::Cell cell(element, it, 0);
 
-    while (runner.KeepRunningIterations(1e6))
-    {
-        cell.Integrate(NuTo::Integrands::TimeDependent::Gradient());
-    }
+    auto Gradient = [&](const NuTo::CellData& cellData, const NuTo::CellIpData& cellIpData) {
+        return integrand.Gradient(cellData, cellIpData, 0);
+    };
+
+    for (auto _ : state)
+        cell.Integrate(Gradient);
 }
+BENCHMARK(NuToPde);
+
+BENCHMARK_MAIN();
```

Comment:
`BENCHMARK_MAIN()` does not need a semicolon. My compiler gives me a pedantic warning here.